modules/imaging_uploader/jsx/columnFormatter.js

```diff
@@ -56,6 +56,11 @@ function formatColumn(column, cell, rowData, rowHeaders) {
   }
 
   if (column === 'Tarchive Info') {
+    if (cell === null || cell === "0") {
```

Comment:
Still it doesn't hurt to check for other falsey values, which `!cell` does. Makes the code less brittle. There isn't a case where you'd want to display a link if cell is `undefined`, `NaN`, or an empty string.